docs/ar/week06/06-2.md

```diff
@@ -0,0 +1,308 @@
+---
+lang-ref: ch.06-2
+lang: ar
+lecturer: Yann LeCun
+title: الشبكات العصبية التكرارية، وحدات البوابات المتكررة، الذاكرة القصيرة المدى المطولة، الانتباه، تسلسل/تسلسل و شبكة الذاكرة.
+authors: Jiayao Liu, Jialing Xu, Zhengyang Bian, Christina Dominguez
+date: 2 مارس 2020
```

Comment:
`translator:` and `translation-date:` are missing.
`data:` field got corrupted.

Reply:
which data is corrupted???